arangod/Transaction/Methods.cpp

```diff
@@ -1035,7 +1041,8 @@ Future<OperationResult> transaction::Methods::insertLocal(
       // be silent.
       // Otherwise, if we already know the followers to replicate to, we can
       // just check if they're empty.
-      if (!followers->empty()) {
+      if (!followers->empty() ||
+          replicationVersion == replication::Version::TWO) {
```

Comment:
Is this really necessary? In case this is a replication 2 db the follower information should now be taken from the state.

Reply:
This will be handled in https://arangodb.atlassian.net/browse/CINFRA-444